core/themes/seven/style.css

```diff
@@ -181,13 +181,20 @@ ul.menu li.expanded {
 /**
  * Tabs.
  */
+@media only screen and (max-width: 600px) {
+  .tabs {
+    clear: both;
+  }
+}
+
```

Comment:
The issue with doing mobile first with this kind of thing is IE8 doesn't support `@media` queries. So we'd need modernizr, a polyfill, or something along those lines to detect when we don't have media queries available so we can make sure those crusty desktop browsers get the styles they need.

That being said, in general I agree, Mobile first media queries are usually the best idea. But considering where Seven is (very desktop-centric) it opens up a can of worms :/